airbyte-integrations/connector-templates/source-configuration-based/source_{{snakeCase name}}/{{snakeCase name}}.yaml.hbs

```diff
@@ -11,7 +11,7 @@ requester:
   name: "\{{ options['name'] }}"
   http_method: "GET"
   authenticator:
-    type: ApiKeyAuthenticator
+    type: BearerAuthenticator
```

Comment:
BearerAuthenticator is probably a better default because it is simpler and very common